import-export-cli/integration/devFirst_test.go

```diff
@@ -182,3 +204,154 @@ func TestImportProjectCreatedPassWhenAPIIsExisted(t *testing.T) {
+	username := superAdminUser
+	password := superAdminPassword
+
+	args := &testutils.InitTestArgs{
+		CtlUser:   testutils.Credentials{Username: username, Password: password},
+		SrcAPIM:   apim,
+		InitFlag:  projectName,
+		OasFlag:   utils.TestOpenAPI3DefinitionPath,
+		ForceFlag: false,
+	}
+
+	testutils.ValidateInitializeProjectWithOASFlag(t, args)
+
+	//Move Image file to created project
+	projectPath, _ := filepath.Abs(projectName)
+	srcPathForImage, _ := filepath.Abs(utils.TestCase2JpegPath)
+	destPathForImage := projectPath + utils.TestCase2DestJpegPathSuffix
+	base.Copy(srcPathForImage, destPathForImage)
+
+	//Import the project with icon image(.jpeg) provided
+	testutils.ValidateImportUpdatePassedWithInitializedProject(t, args)
+
+	testutils.ValidateAPIWithImageIsExported(t, args, utils.DevFirstDefaultAPIName, utils.DevFirstDefaultAPIVersion)
+}
+
+//Import and export API with updated thumbnail and document and assert that
+func TestUpdateDocAndImageOfAPIOfExistingAPI(t *testing.T) {
+	apim := apimClients[1]
+	projectName := "Test"
+	username := superAdminUser
+	password := superAdminPassword
+
+	args := &testutils.InitTestArgs{
+		CtlUser:   testutils.Credentials{Username: username, Password: password},
+		SrcAPIM:   apim,
+		InitFlag:  projectName,
+		OasFlag:   utils.TestOpenAPI3DefinitionPath,
+		ForceFlag: true,
+	}
+	//Initialize the project
+	testutils.ValidateInitializeProjectWithOASFlag(t, args)
+
+	//Move doc file to created project
+	projectPath, _ := filepath.Abs(projectName)
+	srcPathForDoc, _ := filepath.Abs(utils.TestCase2DocPath)
+	destPathForDoc := projectPath + utils.TestCase2DestPathSuffix
+	base.Copy(srcPathForDoc, destPathForDoc)
+
+	//Move Image file to created project
+	srcPathForImage, _ := filepath.Abs(utils.TestCase2JpegPath)
+	destPathForImage := projectPath + utils.TestCase2DestJpegPathSuffix
+	base.Copy(srcPathForImage, destPathForImage)
+
+	//Import the project with Document and image thumbnail
+	testutils.ValidateImportUpdatePassedWithInitializedProject(t, args)
+
+	//Update doc file to created project
+	srcPathForDocUpdate, _ := filepath.Abs(utils.TestCase1DocPath)
+	destPathForDocUpdate := projectPath + utils.TestCase1DestPathSuffix
+	base.Copy(srcPathForDocUpdate, destPathForDocUpdate)
+
+	//Update docMetaData file to created project
+	srcPathForDocMetadataUpdate, _ := filepath.Abs(utils.TestCase1DocMetaDataPath)
+	destPathForDocMetaDataUpdate := projectPath + utils.TestCase1DestMetaDataPathSuffix
+	base.Copy(srcPathForDocMetadataUpdate, destPathForDocMetaDataUpdate)
+
+	//Update icon file to created project
+	srcPathForIcon, _ := filepath.Abs(utils.TestCase2PngPath)
+	destPathForIcon := projectPath + utils.TestCase2DestPngPathSuffix
+	base.Copy(srcPathForIcon, destPathForIcon)
+
+	time.Sleep(1 * time.Second)
+	//Import the project with updated Document and updated image thumbnail
+	testutils.ValidateImportUpdatePassedWithInitializedProject(t, args)
+
+	//Validate that image is been updated
```

Comment:
Shall we change this "is been" to "has been"? There are several comments with grammar mistakes. Can you please correct those? (ex: "a image" -> "an image")

Reply:
Fixed